index.html

```diff
@@ -0,0 +1,738 @@
+                                   <div class="fund-detail">
+                                       <h4>$23M</h4>
+                                       <h5>SERIES A</h5>
+                                   </div>
+                                 </a>
+                                 </div>
+                
+                <ul>
+                 <li><a href="#"><i class="fas fa-tag"></i>Security</a></li>
+                 <li><a href="#"><i class="fas fa-briefcase"></i>August Capital, +2</a></li>
+                </ul>
+                <p><a href="#">General Motors Leads $23M Funding Round for Spring Labs Blockchain Firm <span class="index">on CoinTelegraph</span></a></p>
+                </div>
+                
+                <div class="symphony">
+                        <div class="fund-group">
+                                <a href="#">
+                                   <img src="images/symphony.jpg" alt="symphony">
+                                   <h4 class="fund-group-name">Symphony</h4>
+                                   <div class="fund-detail">
+                                       <h4>$165M</h4>
+                                       <h5>SERIES E</h5>
+                                   </div>
+                                 </a>
+                                 </div>
+                
+                <ul>
+                 <li><a href="#"><i class="fas fa-tag"></i>Messaging, +4</a></li>
+                 <li><a href="#"><i class="fas fa-briefcase"></i>Standard Chartered Bank</a></li>
+                </ul>
+                <p><a href="#">US-based collaboration platform Symphony bags $165m to expand into Asia <span class="index">on Tech in Asia</span></a></p>
+                </div>
+            
+                <div class="buy">
+                        <div class="fund-group">
+                                <a href="#">
+                                   <img src="images/buyandship.jpg" alt="buy and ship">
+                                   <h4 class="fund-group-name">Buyandship</h4>
+                                   <div class="fund-detail">
+                                       <h4>$2.2M</h4>
+                                       <h5>SERIES A</h5>
+                                   </div>
+                                 </a>
+                                 </div>
+                        
+                        <ul>
+                         <li><a href="#"><i class="fas fa-tag"></i>E-commerce, +3</a></li>
+                         <li><a href="#"><i class="fas fa-briefcase"></i>Infinity Venture Capital</a></li>
+                        </ul>
+                        <p><a href="#">Ecommerce shipping startup banks $2.2m in pre-series B round <span class="index">on Tech in Asia</span></a></p>
+                        </div>
+                
+                </div>
+
+
+
+        <div class="group1">
+            <div class="apps group-item">
```

Comment:
This should be a `section` tag instead. Each one of the 8 sections (apps, gear, tech, creative, podium...) must be inside a section tag. Kindly do that with them all.